src/helpers/portfolio/share-portfolio-message.ts

```diff
@@ -17,14 +23,27 @@ const groupMessage = (groupNames, formatMessage) => {
   }
 };
 
+// TODO This will be the form type
+export interface SharePortfolioData {
+  group_uuid: string;
+  permissions: string;
+  groupName: string;
+}
 const sharePorfolioMessage = ({
   shareData,
   initialGroups,
   removedGroups,
   newGroups,
   formatMessage,
   portfolioName
-}) => {
+}: {
+  shareData: SharePortfolioData[];
+  initialGroups: SharePortfolioData[];
+  removedGroups: SharePortfolioData[];
+  newGroups: SharePortfolioData[];
+  formatMessage: (message: MessageDescriptor, values?: AnyObject) => ReactNode;
```

Comment:
There is no official `formatMessage` type? :thinking:

Reply:
No just for all formatters :/
I will add some generic eventually. One we are in the cleanup phase.